### DISCUSSION TITLE
Configuration does not load

### DISCUSSION THREAD

**Discussion by cfsorensen** [UtilitechAS/amsreader-firmware#519] [Q&A] [answered]
Hi,

I have just installed AMS Reader on a new ESP8622 NodeMCU.

I could connect to it over wifi, but when hitting the configuration button I just see a message "Loading configuration", which doesn't not disappear. I have tried to reboot and login from other browsers, but without any success.

<img width="1275" alt="image" src="https://user-images.githubusercontent.com/113860855/230842497-650455b9-1090-4b76-944e-de4e44d98a42.png">

Any suggestions for how to fix this will be highly valued.

**Comment by gskjold** [comment]
This suggests that there are some data in the config that is breaking the JSON file sent to the GUI. Could you please go to URL /configuration.json on your device and check for strange characters.

Config backup and restore on the system page may fix your problem.

**Reply by cfsorensen** [reply]
Hi,

I see this in the json, and that is with totally fresh install on new board, so nothing can really be backed up and restored.

{"version":"v2.2.13","g":{"t":"Europe/Oslo","h":"ams-XXXXXX,”s”:0,”u”:””,”p”:””},”m”:{“b”:0,”p”:0,”I”:false,"d":2,"f":40,"r":0,"e":{"e":false,"k":"00000000000000000000000000000000","a":"00000000000000000000000000000000"},"m":{"e":false,"w":0.000,"v":0.000,"a":0.000,"c":0.000}},"t":{"t":[5,10,15,20,25,50,75,100,150,65535],"h":3},"w":{"s":"Wifi","p":"***","w":20.5,"z":1,"a":false},"n":{"m":"dhcp","i":"","s":"","g":"","d1":"","d2":"","d":true,"n1":"pool.ntp.org","h":true},"q":{"h":"","p":1883,"u":"","a":"","c":"","b":"","m":0,"s":{"e":false,"c":false,"r":false,"k":false}},"p":{"e":false,"t":"","r":"","c":"","m":1.000,"f":16.3830000000},"d":{"s":false,"t":false,"l":5},"i":{"h":{"p":16,"u":true},"a":null,"l":{"p":2,"i":true},"r":{"r":null,"g":null,"b":null,"i":true},"t":{"d":null,"a":null},"v":{"p":null,"o":0.00,"m":1.000,"d":{"v":0,"g":0},"b":0.0}},"u":{"i":1,"e":2,"v":2,"a":2,"r":0,"c":1,"t":2,"p":2,"d":1,"m":1,"s":2},"o":{"e":0,"c":0,"u1":0,"u2":0,"u3":0},"h":{"t":"������������������������������������������������������������������������������������������������������������������������������������������������pool.ntp.org","h":"��������������������������������������������������������������������������������pool.ntp.org","n":"����������������pool.ntp.org"}}

**Comment by tubbelino** [comment]
I have the same problem using two types of Wemos D1 mini boards, one original and one clone. But the problem is the same.
After version 2.2.12 it is not possible to access the configuration page if the unit was flashed on to a new esp8266 board.
The workaround i have found is to flash version 2.2.11 and then update to 2.2.13, this works without problem.
Otherwise AMS Reader is working perfectly!

**Reply by rogerthn2019** [reply]
Thanks!

**Reply by gskjold** [reply, marked as answer]
This was an issue we had that was fixed in 2.2.14
